src/utils/daos_hdlr.c

```diff
@@ -3670,27 +3383,20 @@ cont_clone_hdlr(struct cmd_args_s *ap)
 				    &ca.dst_coh, &dst_cont_info, NULL);
 		if (rc == 0) {
 			fprintf(ap->errstream,
-				"This destination container already "
-				"exists, please provide a destination "
-				"container uuid that does not exist already, "
-				"or provide an existing pool or new UNS path "
-				"of the form:\n\t--dst </$pool> | "
-				"<path/to/uns>\n");
+				"This destination container already exists, please provide a "
```

Comment:
Grammar nitpick
```suggestion
				"This destination container already exists; please provide a "
```
Or
```suggestion
				"This destination container already exists. Please provide a "
```